.github/workflows/pull_request_tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
Why this change?

Reply:
Uh, very weird. I didn't really change this file. But maybe during some merge this must've happened. 
I'll revert this change to os: [ubuntu-latest, macos-latest, windows-latest]